packetbeat/docs/fields.asciidoc

```diff
@@ -89,32 +89,33 @@ Indicates if event is last event in flow. If final is false, the event reports a
 Internal flow id based on connection meta data and address.
 
 
-==== mac_source
+==== vlan
 
-Source MAC address as indicated by first packet seen for the current flow.
+Innermost VLAN address used in network packets.
 
 
-==== mac_dest
+==== outter_vlan
```

Comment:
oh, typo. Just figured  it must say 'outer', not 'outter'. I'm afraid this error is all over the place.